client/jmx-adapter/pom.xml

```
@@ -37,7 +37,12 @@
             <artifactId>testng</artifactId>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
+            <groupId>org.awaitility</groupId>
+            <artifactId>awaitility</artifactId>
+            <version>4.0.0</version>
```

there is a 4.2.0 https://github.com/awaitility/awaitility/releases/tag/awaitility-4.2.0
```suggestion
            <version>4.2.0</version>
```